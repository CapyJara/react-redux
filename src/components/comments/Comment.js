import React from 'react';
import PropTypes from 'prop-types';

function Comment({ comment }) {
  return (
  <>
    <h3>{comment.title}</h3>
    <p>{comment.comment}</p>
  </>
  );
}

Comment.propTypes = {
  comment: PropTypes.shape({
    title: PropTypes.string.isRequired,
    comment: PropTypes.string.isRequired
  }).isRequired,
};

export default Comment;
