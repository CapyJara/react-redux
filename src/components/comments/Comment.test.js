import React from 'react';
import { shallow } from 'enzyme';
import Comment from './Comment';

describe('Comment component', () => {
  it('renders a Comment', () => {
    const comment = {
      title: 'title of my comment',
      comment: 'this is my comment'
    };

    const wrapper = shallow(<Comment comment={comment} />);
    expect(wrapper).toMatchSnapshot();
  });
});
