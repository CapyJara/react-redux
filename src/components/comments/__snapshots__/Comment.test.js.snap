// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Comment component renders a Comment 1`] = `
<Fragment>
  <h3>
    title of my comment
  </h3>
  <p>
    this is my comment
  </p>
</Fragment>
`;
